src/core/inductive.ml

```diff
@@ -0,0 +1,9 @@
+open Timed
+open Terms
+
+type inductive =
```

Comment:
This type should be define in sign.ml I think because you need to access some builtins in inductive.ml.
Thanks to rename ind_constructors into ind_cons (this is shorter and clear enough).

Reply:
Yes, I think too.